packages/api-kit/src/health.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, NextFunction } from 'express';  
+import { performance } from 'perf_hooks';  
+
+export function healthCheckMiddleware(req: Request, res: Response, next: NextFunction) {  
+  if (req.path === '/healthz') {  
+    const start = performance.now();  
```

Comment:
Is this a good idea for production? Actually I don't know if it affect performance.